generic-tabs/GenericTabs.js

```diff
@@ -70,11 +70,29 @@ export class GenericTabs extends SelectedMixin(BatchingElement) {
 
   constructor() {
     super();
+
+    // Check if slots were setup
+    // If not, assume heading structure and preprocess light DOM
+    if (!this.querySelector('[slot]')) {
+      const unformattedEls = [...this.children];
+      unformattedEls.forEach(node => {
+        if (['h1', 'h2', 'h3', 'h4', 'h5', 'h6'].includes(node.tagName.toLowerCase())) {
```

Comment:
Nice preprocessing. What would think about inferring headings from their instance?

```suggestion
        if (node instanceof HTMLHeadingElement) {
```

Reply:
Way better! I never use `instanceof` so I forgot it existed.